### DISCUSSION TITLE
Error when loading `this.libQueryEnginePath`

### DISCUSSION THREAD

**Discussion by erikmellum** [prisma/prisma#17912] [Q&A]
After upgrading from 4.6.0 to 4.9.0 we began getting this error. Wondering if anyone has a suggestion on how to resolve?

![Screenshot2023_02_13_080800](https://user-images.githubusercontent.com/6475033/218535320-eda85ae2-121c-44ae-a11b-6118bc4c01fb.jpg)

**Comment by nurul3101** [comment]
Hey @erikmellum 👋 
When exactly do you get this error?

Do you get it when invoking any prisma commands? Also, does the error go away if you downgrade to 4.6.0 version?

**Reply by erikmellum** [reply]
Thank you for your time @nurul3101.

This is happening when we run our jest test suite, specifically the integration tests. 

Let me downgrade again to make sure, but my team indicates that the tests were working before I upgraded to prisma 4.9.0.

**Reply by nurul3101** [reply]
Sure, let me know if the issue gets fixed by downgrading. It would allows us to identify if this issue is related to a particular prisma version.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
